src/main.py

```diff
@@ -58,8 +58,17 @@ def standby():
 
     Once launch is detected, this method will return.
     """
-    # TODO: Implement this state
-    pass
+  # TODO: Implement this state
+    gpio = GPIOReader(False)
+    RetrieveData = gpio.retrieveData()
```

Comment:
Variables should be created with lower case names

```suggestion
    retrieve_data = gpio.retrieveData()
```

The method name on gpio should also be lower sake case (`retrieve_data()`) however it is external to this PR, so I'd leave it for now.